packages/data-wizard/src/dataset/utils.ts

```diff
@@ -61,3 +62,32 @@ export const stringify = (value: any) =>
     ?.replace(/\}"/g, ' }') || 'undefined';
 
 export const getStringifyLength = (value: any) => stringify(value)?.length;
+
+/**
+ * Convert data to specified data type.
+ * @param datum
+ * @param type
+ * @returns converted data
+ */
+export const convertDataType = (data: any, type: Extra['columnTypes'][number]) => {
+  try {
+    if (type === 'string' && !isString(data)) {
+      return `${data}`;
+    }
+    if (type === 'boolean' && !isBoolean(data)) {
+      return Boolean(data);
+    }
+    if (type === 'null' && !isNull(data)) {
+      return null;
+    }
+    if ((type === 'integer' || type === 'float') && !isNumber(data)) {
+      return +data;
+    }
+    if (type === 'date' && !isDateString(`${data}`)) {
+      return new Date(data);
+    }
+  } catch (error) {
```

Comment:
Support data transformation, avoid cases that always output string for number-type columns